src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="field" data-qa="card">
+      <div class="field__logo">
+        <img 
+          src="..\src\images\imac.jpeg" 
+          alt="field__logo" 
+          class="logo"
+        >
+      </div>
+      <p class="field__text-name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="field__text-product-code">
+        Код товара: 195434
+      </p>
+      <div class="review-container">
+        <div class="review-container__stars">
+          <div class="star star--1"></div>
+          <div class="star star--2"></div>
+          <div class="star star--3"></div>
+          <div class="star star--4"></div>
+          <div class="star star--5"></div>
+        </div>
```

Comment:
```suggestion
        <div class="review-container__stars stars stars--4">
          <div class="stars__star"></div>
          <div class="star"></div>
          <div class="star"></div>
          <div class="star"></div>
          <div class="star"></div>
        </div>
```
Используй код из задачи stars

Класс-модификатор должен быть только у контейнера, у детей только один класс